### ISSUE TITLE
Updated amro.omp.json User Icon Not Working with Meslo Nerd Font

### ISSUE THREAD

**Issue by SuperJC710e** [JanDeDobbeleer/oh-my-posh#3826]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

The unicode changed from `\uf508` to `\ueb99`, which is blank in the suggested Meslo Font.

With the New Text Renderer "AtlasEngine":
![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/13032430/d359a335-9b5c-4c53-91c5-62953c5bc886)

Without:
![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/13032430/d076bb3a-ebf5-4497-9610-3e32d899b35e)

Creating an override and setting it back to `\uf508` fixes the issue, but would prefer to just point at the original theme, to always get the latest updates.

```json
{
  "$schema": "https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/schema.json",
  "blocks": [
    {
      "alignment": "left",
      "segments": [
        {
          "foreground": "#45F1C2",
          "style": "plain",
          "template": "\uf508 {{ .UserName }} on",
          "type": "session"
        },
        {
          "foreground": "#0CA0D8",
...
```

### Theme

Amro

### What OS are you seeing the problem on?

Windows

### Which shell are you using?

powershell

### Log output

```Shell
Version: 16.2.2

Shell: pwsh (7.3.4)

Segments:

ConsoleTitle(false)                       -   0 ms -
session(true)                             -   1 ms -  <user> on
path(true)                                -   2 ms -   ~
git(false)                                -  32 ms -
root(false)                               -   0 ms -
text(true)                                -   0 ms - #

Run duration: 52.7571ms

Cache path: C:\Users\<user>\AppData\Local\oh-my-posh

Config path: C:\Users\<user>\amro-override.omp.json

Logs:

[DEBUG] 16:57:13.785 shell.go:Getenv:395 → NO DATA
[TRACE] 16:57:13.785 shell.go:Getenv(TERM_PROGRAM) - 1.966ms
[DEBUG] 16:57:13.785 shell.go:Getenv:395 → C:\Users\<user>\AppData\Local
[TRACE] 16:57:13.785 shell.go:Getenv(LOCALAPPDATA) - 0s
[TRACE] 16:57:13.785 shell_windows.go:CachePath() - 0s
[DEBUG] 16:57:13.786 shell.go:Getenv:395 → C:\Users\<user>\amro-override.omp.json
[TRACE] 16:57:13.786 shell.go:Getenv(POSH_THEME) - 0s
[DEBUG] 16:57:13.797 shell.go:Shell:642 → process name: pwsh.exe
[TRACE] 16:57:13.797 shell.go:Shell() - 11.0308ms
[TRACE] 16:57:13.797 shell.go:resolveConfigPath() - 11.0308ms
[TRACE] 16:57:13.797 shell.go:Init() - 13.9975ms
[TRACE] 16:57:13.797 shell.go:Flags() - 0s
[TRACE] 16:57:13.798 config.go:loadConfig() - 1.0048ms
[TRACE] 16:57:13.798 shell.go:Flags() - 0s
[DEBUG] 16:57:13.798 shell.go:Getenv:395 → NO DATA
[TRACE] 16:57:13.798 shell.go:Getenv(OMP_CACHE_DISABLED) - 0s
[TRACE] 16:57:13.799 shell_windows.go:WindowsRegistryKeyValue(HKEY_CURRENT_USER\Software\Microsoft\Windows\DWM\ColorizationColor) - 0s
[TRACE] 16:57:13.799 shell.go:GOOS() - 0s
[TRACE] 16:57:13.799 shell.go:GOOS() - 0s
[TRACE] 16:57:13.799 shell.go:GOOS() - 0s
[TRACE] 16:57:13.799 shell.go:GOOS() - 0s
[TRACE] 16:57:13.799 shell.go:GOOS() - 0s
[TRACE] 16:57:13.799 shell.go:GOOS() - 0s
[TRACE] 16:57:13.799 shell.go:GOOS() - 0s
[TRACE] 16:57:13.799 shell.go:GOOS() - 0s
[TRACE] 16:57:13.799 shell.go:GOOS() - 0s
[TRACE] 16:57:13.799 shell.go:GOOS() - 0s
[TRACE] 16:57:13.799 shell.go:GOOS() - 0s
[TRACE] 16:57:13.799 shell.go:GOOS() - 0s
[TRACE] 16:57:13.799 shell.go:GOOS() - 0s
[TRACE] 16:57:13.799 shell.go:GOOS() - 0s
[TRACE] 16:57:13.799 shell.go:GOOS() - 0s
[TRACE] 16:57:13.799 shell.go:GOOS() - 0s
[TRACE] 16:57:13.799 shell.go:GOOS() - 0s
[TRACE] 16:57:13.799 shell.go:GOOS() - 0s
[TRACE] 16:57:13.799 shell.go:GOOS() - 0s
[DEBUG] 16:57:13.799 shell_windows.go:WindowsRegistryKeyValue:206 → ColorizationColor(DWORD): 0xC40078D7
[TRACE] 16:57:13.799 shell.go:Shell() - 0s
[DEBUG] 16:57:13.799 shell.go:Getenv:395 → 7.3.4
[TRACE] 16:57:13.799 shell.go:Getenv(POSH_SHELL_VERSION) - 0s
[DEBUG] 16:57:13.799 debug.go:PrintDebug:22 → Segment: Title
[TRACE] 16:57:13.799 shell.go:Flags() - 0s
[DEBUG] 16:57:13.799 block.go:Debug:230 → Segment: session
[TRACE] 16:57:13.799 shell.go:GOOS() - 0s
[DEBUG] 16:57:13.800 shell.go:Pwd:425 → C:\Users\<user>
[TRACE] 16:57:13.800 shell.go:Pwd() - 877.3µs
[DEBUG] 16:57:13.800 shell_windows.go:56 → C:\Users\<user>
[TRACE] 16:57:13.800 shell.go:GOOS() - 0s
[DEBUG] 16:57:13.800 shell.go:Getenv:395 → NO DATA
[TRACE] 16:57:13.800 shell.go:Getenv(SSH_CONNECTION) - 0s
[DEBUG] 16:57:13.800 shell.go:Getenv:395 → NO DATA
[TRACE] 16:57:13.800 shell.go:Getenv(SSH_CLIENT) - 0s
[TRACE] 16:57:13.800 shell_windows.go:Root() - 0s
[TRACE] 16:57:13.800 shell.go:Shell() - 0s
[TRACE] 16:57:13.800 shell.go:ErrorCode() - 0s
[TRACE] 16:57:13.800 shell_windows.go:IsWsl() - 0s
[TRACE] 16:57:13.800 shell.go:Pwd() - 0s
[DEBUG] 16:57:13.800 shell_windows.go:56 → C:\Users\<user>
[TRACE] 16:57:13.800 shell.go:GOOS() - 0s
[TRACE] 16:57:13.800 shell.go:GOOS() - 0s
[TRACE] 16:57:13.800 shell.go:GOOS() - 0s
[TRACE] 16:57:13.800 shell.go:GOOS() - 0s
[TRACE] 16:57:13.800 shell.go:GOOS() - 0s
[TRACE] 16:57:13.800 shell.go:GOOS() - 0s
[TRACE] 16:57:13.800 shell.go:GOOS() - 0s
[TRACE] 16:57:13.800 shell.go:GOOS() - 0s
[TRACE] 16:57:13.800 shell.go:GOOS() - 0s
[DEBUG] 16:57:13.800 shell.go:User:546 → <user>
[TRACE] 16:57:13.800 shell.go:User() - 0s
[DEBUG] 16:57:13.801 shell.go:Host:558 → BRI-108984L
[TRACE] 16:57:13.801 shell.go:Host() - 820µs
[TRACE] 16:57:13.801 shell.go:GOOS() - 0s
[DEBUG] 16:57:13.801 shell.go:Getenv:395 → NO DATA
[TRACE] 16:57:13.801 shell.go:Getenv(SHLVL) - 0s
[TRACE] 16:57:13.801 shell.go:TemplateCache() - 820µs
[TRACE] 16:57:13.801 shell.go:TemplateCache() - 0s
[TRACE] 16:57:13.801 shell.go:Shell() - 0s
[DEBUG] 16:57:13.801 block.go:Debug:230 → Segment: path
[TRACE] 16:57:13.801 shell.go:Pwd() - 0s
[DEBUG] 16:57:13.801 shell_windows.go:56 → C:\Users\<user>
[TRACE] 16:57:13.801 shell.go:GOOS() - 0s
[TRACE] 16:57:13.801 shell.go:Pwd() - 0s
[TRACE] 16:57:13.801 shell.go:Shell() - 0s
[TRACE] 16:57:13.801 shell.go:Flags() - 0s
[DEBUG] 16:57:13.802 properties.go:GetBool:22 → mapped_locations_enabled: true
[DEBUG] 16:57:13.802 properties.go:GetString:28 → 
[DEBUG] 16:57:13.802 properties.go:GetString:28 → 
[DEBUG] 16:57:13.802 shell_windows.go:56 → C:\Users\<user>
[TRACE] 16:57:13.802 shell.go:GOOS() - 0s
[TRACE] 16:57:13.802 shell.go:GOOS() - 0s
[DEBUG] 16:57:13.802 properties.go:GetString:28 → ~
[DEBUG] 16:57:13.802 properties.go:GetKeyValueMap:46 → mapped_locations: map[]
[TRACE] 16:57:13.802 shell.go:PathSeparator() - 0s
[TRACE] 16:57:13.802 shell.go:GOOS() - 0s
[TRACE] 16:57:13.802 shell.go:GOOS() - 0s
[TRACE] 16:57:13.802 shell.go:GOOS() - 0s
[TRACE] 16:57:13.802 shell.go:GOOS() - 0s
[TRACE] 16:57:13.802 shell.go:GOOS() - 0s
[TRACE] 16:57:13.802 shell.go:GOOS() - 0s
[TRACE] 16:57:13.802 shell.go:GOOS() - 0s
[TRACE] 16:57:13.802 shell.go:PathSeparator() - 0s
[TRACE] 16:57:13.802 shell.go:PathSeparator() - 0s
[TRACE] 16:57:13.802 shell.go:GOOS() - 0s
[TRACE] 16:57:13.802 shell.go:GOOS() - 0s
[TRACE] 16:57:13.802 shell.go:GOOS() - 0s
[TRACE] 16:57:13.802 shell.go:PathSeparator() - 0s
[TRACE] 16:57:13.802 shell.go:GOOS() - 0s
[TRACE] 16:57:13.802 shell.go:GOOS() - 0s
[TRACE] 16:57:13.802 shell.go:GOOS() - 0s
[TRACE] 16:57:13.802 shell.go:PathSeparator() - 0s
[TRACE] 16:57:13.802 shell.go:GOOS() - 0s
[TRACE] 16:57:13.802 shell.go:GOOS() - 0s
[TRACE] 16:57:13.802 shell.go:GOOS() - 0s
[TRACE] 16:57:13.802 shell.go:PathSeparator() - 0s
[TRACE] 16:57:13.802 shell.go:Pwd() - 0s
[TRACE] 16:57:13.802 shell_windows.go:IsWsl() - 0s
[TRACE] 16:57:13.802 shell.go:StackCount() - 0s
[DEBUG] 16:57:13.803 win32_windows.go:isWriteable:299 → not current user or in group
[DEBUG] 16:57:13.803 win32_windows.go:isWriteable:303 → current user is member of S-1-5-32-544
[DEBUG] 16:57:13.803 win32_windows.go:isWriteable:311 ↓
    WRITE_DAC
    WRITE_OWNER
    SYNCHRONIZE
    DELETE
    READ_CONTROL
[DEBUG] 16:57:13.803 win32_windows.go:isWriteable:313 → user has write access
[TRACE] 16:57:13.803 shell_windows.go:DirIsWritable() - 1.1401ms
[TRACE] 16:57:13.803 shell.go:TemplateCache() - 0s
[TRACE] 16:57:13.803 shell.go:TemplateCache() - 0s
[TRACE] 16:57:13.803 shell.go:Shell() - 0s
[DEBUG] 16:57:13.803 block.go:Debug:230 → Segment: git
[TRACE] 16:57:13.803 shell.go:Pwd() - 0s
[DEBUG] 16:57:13.803 shell_windows.go:56 → C:\Users\<user>
[TRACE] 16:57:13.803 shell.go:GOOS() - 0s
[TRACE] 16:57:13.803 shell.go:GOOS() - 0s
[DEBUG] 16:57:13.834 shell.go:CommandPath:597 → C:\Program Files\Git\cmd\git.exe
[TRACE] 16:57:13.834 shell.go:CommandPath(git.exe) - 31.5545ms
[TRACE] 16:57:13.834 shell.go:HasCommand(git.exe) - 31.5545ms
[TRACE] 16:57:13.834 shell.go:Pwd() - 0s
[ERROR] 16:57:13.836 shell.go:HasParentFilePath:730 → CreateFile .git: The system cannot find the file specified.
[TRACE] 16:57:13.836 shell.go:HasParentFilePath(.git) - 1.0452ms
[DEBUG] 16:57:13.836 properties.go:GetBool:22 → fetch_bare_info: false
[TRACE] 16:57:13.836 shell.go:Flags() - 0s
[DEBUG] 16:57:13.836 block.go:Debug:230 → Segment: root
[TRACE] 16:57:13.836 shell.go:Pwd() - 0s
[DEBUG] 16:57:13.836 shell_windows.go:56 → C:\Users\<user>
[TRACE] 16:57:13.836 shell.go:GOOS() - 0s
[TRACE] 16:57:13.836 shell_windows.go:Root() - 0s
[DEBUG] 16:57:13.836 block.go:Debug:230 → Segment: text
[TRACE] 16:57:13.836 shell.go:Pwd() - 0s
[DEBUG] 16:57:13.836 shell_windows.go:56 → C:\Users\<user>
[TRACE] 16:57:13.836 shell.go:GOOS() - 0s
[TRACE] 16:57:13.836 shell.go:TemplateCache() - 0s
[TRACE] 16:57:13.836 shell.go:Shell() - 0s
[DEBUG] 16:57:13.836 shell.go:Getenv:395 → C:\Users\<user>\AppData\Local
[TRACE] 16:57:13.836 shell.go:Getenv(LOCALAPPDATA) - 0s
[TRACE] 16:57:13.836 shell_windows.go:CachePath() - 0s
[TRACE] 16:57:13.836 shell.go:Flags() - 0s
```

**Comment by JanDeDobbeleer** [comment]
@SuperJC710e did you update your font to V3?

**Comment by SuperJC710e** [comment]
@JanDeDobbeleer Thanks for the quick response. 

I think I did, I ran the new command (just noticed it this round): `oh-my-posh font install` and selected the `Meslo` font in an Elevated shell. I closed all open terminals and reopened it after. Would I need to anything more, like rebooting the system?

**Comment by JanDeDobbeleer** [comment]
@SuperJC710e if `\uf508` still works, the font isn't loaded indeed. But I'll check real quick myself too.

**Comment by JanDeDobbeleer** [comment]
@SuperJC710e it's definitely the font as the [rendering works with a V3 font](https://ohmyposh.dev/docs/themes#amro). So the icon is available.

**Comment by cupsos** [comment]
@JanDeDobbeleer  I have same issue. It looks like `oh-my-posh install` and Nerd Font v3 naming problem.

`oh-my-posh install` command on windows, it only install "windows compatible" fonts.
https://github.com/JanDeDobbeleer/oh-my-posh/blob/865374a0f557019028ac3cd9129eb6b8672350bb/src/font/install.go#L67-L75

But from Nerd Font v3
>https://github.com/ryanoasis/nerd-fonts/releases/tag/v3.0.0
>There are no Windows Compatible fonts anymore. All fonts work on all platforms.

@SuperJC710e  Try remove Meslo font and reinstall it manually (without `oh-my-posh install`).
https://github.com/ryanoasis/nerd-fonts/releases/download/v3.0.0/Meslo.zip
Also you may need to change font `MesloLGM NF` to `MesloLGM Nerd Font`.

**Comment by JanDeDobbeleer** [comment]
@cupsos ah! That's a change I wasn't aware off. I'll adjust that!

**Comment by JanDeDobbeleer** [comment]
@cupsos https://github.com/JanDeDobbeleer/oh-my-posh/pull/3828

**Comment by SuperJC710e** [comment]
> @SuperJC710e it's definitely the font as the [rendering works with a V3 font](https://ohmyposh.dev/docs/themes#amro). So the icon is available.

Thanks @JanDeDobbeleer!

So I did eventually try a reboot (it's my work system, and had too many things going on to reboot earlier), and it didn't fix things. 

So then I tried as @cupsos suggested. I downloaded the `Meslo.zip` from the github repo release 3.0.0 (or whatever it was), extracted and installed them all as system fonts. I then selected `MesloLGM Nerd Font Mono` as my default font, and restored my profile to use the package provided version...and voila! it works!

So your changes to the font install utility should catch it too! (Not sure if it will be able to force install the newer versions, but I didn't look over the code...)

Anyway, thanks again for the prompt responses and checking out my little bug report! :)
